### ISSUE TITLE
import-scope false positive with `lodash/fp`

### ISSUE THREAD

**Issue by SpainTrain** [wix/eslint-plugin-lodash#134]
When using the `member` import style with the `import-scope` rule, imports from `lodash/fp` are incorrectly marked as errors.  E.g., 

```js
import {
    filter as fpFilter,
    map as fpMap,
} from 'lodash/fp'
```

reports `Import members from the full lodash module`

IIRC, `fp` will be moved to a different module in the next lodash major version, but for now I think this is incorrect behavior?  If agreed, I can probably work up a PR to fix it.

**Comment by ganimomer** [comment]
Fix released in `v2.3.5`.
Thanks for reporting and fixing it!
